src/main/java/mate/academy/service/AuthenticationService.java

```diff
@@ -11,6 +11,11 @@ public class AuthenticationService {
      * Return false in any other cases.
      */
     public boolean login(String email, String password) {
+        UserService userService = new UserService();
```

Comment:
let's not create a new service each time we call a `login()` method, and make it an instance level variable of `AuthenticationService`